src/masonite/providers/FrameworkProvider.py

```diff
@@ -1,22 +1,35 @@
 from email.mime import application
-from ..foundation import response_handler
+import time
+
 from ..request import Request
 from ..response import Response
-import time
+from .Provider import Provider
 
+from ..presets.PresetsCapsule import PresetsCapsule
+from ..presets import Tailwind, Vue, React, Bootstrap
 
-class FrameworkProvider:
+
+class FrameworkProvider(Provider):
     def __init__(self, application):
         self.application = application
 
     def register(self):
-        pass
+        # @M5 remove this and add PresetsProvider in default project
```

Comment:
For now I put PresetsProvider code here. @josephmancuso does this work for you or do you have a better idea ?